src/main/java/org/cascadebot/cascadebot/CascadeBot.java

```diff
@@ -318,4 +350,11 @@ public long getUptime() {
         return System.currentTimeMillis() - startupTime;
     }
 
+    public Jedis getRedisClient() {
+        return redisClient;
+    }
+
+    public MessageReceivedRunnable getMessageReceivedRunnable() {
+        return messageReceivedRunnable;
+    }
```

Comment:
Perhaps we could make the runnable a singleton instead of being another thing contained with the main file